core/internal/cltest/mocks.go

```diff
@@ -308,6 +309,18 @@ func MustRandomUser(t testing.TB) sessions.User {
 	return r
 }
 
+func NewUserWithSession(t testing.TB, orm sessions.ORM) sessions.User {
```

Comment:
Out of scope, but this constructor set is pretty confusing, with New/Create/nothing:
- `MustRandomUser`
- `CreateUser*`
- `MustNewUser`

Reply:
I cleaned this up :) Only MustRandomUser and NewUserWithSession remain now.